centos/nap/Dockerfile

```diff
@@ -48,8 +48,14 @@ RUN curl -k -sS -L ${CONTROLLER_URL} > install.sh \
 
 FROM agent-installer as nap-installer
 # Install nginx-app-protect
-RUN yum -y install app-protect \
-  && sed -i "6 a load_module modules/ngx_http_app_protect_module.so;" /etc/nginx/nginx.conf
+ARG NGINX_PLUS_VERSION=23
+RUN if [ "$NGINX_PLUS_VERSION" -lt 23 ] ; \
```

Comment:
When user specifies not supported nginx plus release version by nap then make it as hard error.